code/src/layouts/header/components/header-toolbar/HeaderToolbar.tsx

```diff
@@ -41,6 +43,10 @@ const HeaderToolbar = () => {
   const userRole = useUserRoleSelector();
   const dispatch = useAppDispatch();
 
+  const [fetch, { data }] = useLazyGetCartItemsQuery();
```

Comment:
I suggest using another name to avoid collisions with native javascript `fetch`

Reply:
Thank you. Changed it to `fetchCartItems`